autogen/cluster_regional.tf

```diff
@@ -56,6 +56,11 @@ resource "google_container_cluster" "primary" {
     }
   }
 
+  network_policy {
+    provider = "CALICO"
+    enabled  = true
```

Comment:
This seems to hard code network policy to `true. Shouldn't we only enable it if `network_policy` variable is true?

Reply:
Sorry, I'll fix it. This was a typo during the edition.

Reply:
@morgante I uploaded the fix.